### DISCUSSION TITLE
maxhotkey error

### DISCUSSION THREAD

**Discussion by gunnho** [256shadesofgrey/eso-light-attack-weave#23] [General]
i enabled key5 and set my mouse button to 5 and dropped a spammable skill in slot 5, so as i am strafing holding down the mousebutton i can la weave perfectly and am happy with this ahk macro, however if i accidentally press any other key while the mouse button is pressed, i get a popup stating maxhotkey pressed, do i want to continue yes/no, sometimes the message disappears under the screen and now no buttons work at all until i find the popup and click yes

how can i get rid of that?

**Comment by 256shadesofgrey** [comment]
So you are holding down 5, and when you use any other skill (say 4) while holding down 5, you get this error, did I get it right?

Also, what exactly does the error message say? Ideally, could you make a screenshot?

Thank you.

**Reply by gunnho** [reply]
![maxhotkey](https://github.com/256shadesofgrey/eso-light-attack-weave/assets/47189749/6a1188d9-7b3f-4b3b-8a2c-f036e5e126e9)

**Reply by 256shadesofgrey** [reply]
Try adding `#MaxHotkeysPerInterval 9999` as the first line in the script. Does it fix it?

**Reply by gunnho** [reply]
this was during a quick fight, while holding down 5, pressing RMB for shield and pressing 1 or 2

**Reply by gunnho** [reply]
yes, this fixed it perfectly, thanks! i can now block while la weaving, and cast unstable wall. tysm!
